infrastructure/db/handler.js

```diff
@@ -1,253 +1,208 @@
+    }
 }
-
-exports.addTeamMemeber = async(request) => {
-    models.teammember.create({ emailAddress : request.emailAddress, 
-        password : request.password });
+/**
+* Creates Teammember
+* @param {JSON} member Details of the new members
+* @return Nothing.
+*/
+const createTeamMemeber = async(member) => {
+    models.teammember.create({ emailAddress : member.emailAddress, 
+        password : member.password });
 
 }
 
 /**
 * Finds all the Teammembers of specific board
 * @param {int} boardId id of the wanted board
-* @return {Promise} promise, with json response
+* @return {JSON} json, with board's teammembers details
 */
-exports.getBoardTeammembers = async(boardId) => {
-    
-    return new Promise((resolve) => {
-
-        models.board.findByPk(boardId).then((board) => {
-            board.getTeammembers().then(users => {
-
-                let resp = { Teammembers : new Array() };
-                for(let i = 0;i < users.length;i++) {
-                    resp.Teammembers[i] = {
-                        id : users[i].id,
-                        emailAddress : users[i].emailAddress,
-                        createdAt : users[i].createdAt,
-                        updatedAt : users[i].updatedAt
-                    }
-                }
-                resolve(resp);
-
-            });
-        });
-
-    });
+const getBoardTeammembers = async(boardId) => {
+    try {
+        const board = await getBoard(boardId);
+        if(board != null) {
+            return await board.getTeammembers();
+        } else {
+            return [];
+        }
+    } catch(err) {
+        throw Error(err);
+    }
 }
 /**
 * Finds all the Groups of specific board
 * @param {int} boardId id of the wanted board
-* @return {Promise} promise, with json response
+* @return {JSON} json with all groups of requested board
 */
-exports.getGroups = async(boardId) => {
-
-    return new Promise((resolve) => {
-
-        models.group.findAll({ where : { boardId : boardId } }).then(groups => {
-
-            let resp = { groups : new Array() }; //Builds array of all the groups
-            for(let i = 0;i < groups.length;i++) {
-                resp.groups[i] = {
-                    id : groups[i].id,
-                    label : groups[i].label,
-                    updatedAt : groups[i].updatedAt
-                }
-            }
-            resolve(resp);
-
-        });
-    });
+const getGroups = async(boardId) => {
+    try {
+        return await models.group.findAll({ where : { boardId : boardId } });
+    } catch(err) {
+        throw Error(err);
+    }
 }
 
-exports.addGroup = async(request) => {
+/**
+* Checks if a Teammember is a member of a certain board
+* @param {int} boardId id of the wanted board
+* @param2 {int} teammemberId id of the wanted teammember
+* @return {Boolean} True - if a member of the board, else False
+*/
+const isBoardTeammember = async(boardId, teammemberId) => {
+    const sequelize = await getSequelize();
```

Comment:
In general I think that when you use an ORM (object relational mapping) tooling such as Sequelize you shouldn't use the sequelize.query method as it's very unclear & not readable to read RAW SQL query instead of the sequelize API. Let's discuss it together (?) like what are you trying to do exactly and perhaps we can express it via the sequelize API itself?